content/blog/posts/student-story-doublej472.rst

```diff
@@ -0,0 +1,42 @@
+On Leaving the Open Source Lab, Jonathan Frederick
+--------------------------------------------------
+:author: Jonathan Frederick
+:date: 2019-09-25
+:img: doublej472-story.jpg
+:slug: student-story-doublej472
+:tag: featured-stories
+:tags: student-stories
+
+My first Linux operating system was Ubuntu 10.04, when I was still young, and it was my first experience with open
+source software, which I didn't fully appreciate at the time. I just wanted an operating system to work on my Dell
+hand-me-down computer and I didn't want to pay for windows, and Linux was the first thing I found. Over the next few
+years I started to really appreciate open source software and realized it's importance on the internet and for myself,
+since it was one of the only things that allows me to have full control over my devices.
```

Comment:
```suggestion
since it was one of the only things that allow me to have full control over my devices.
```

Reply:
`allowed` is better, since this sentence is past-tense

Reply:
Correct! :100: